File: src/sdk-reference/cpp/1/auth/update-my-credentials/snippets/update-my-credentials.cpp
try {
  kuzzle->auth->login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
  kuzzle->auth->updateMyCredentials("local", "{\"username\":\"foo\",\"password\":\"bar\",\"other\":\"value\"}");

  std::cout << "Credentials successfully updated" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/update-my-credentials/snippets/update-my-credentials.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  kuzzle.auth.updateMyCredentials(
    "local",
    @"{""username"":""foo"",""password"":""bar"",""other"":""value""}");

  Console.WriteLine("Credentials successfully updated");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
